build_tools/index_generation_s3_tar.py

```diff
@@ -92,6 +103,8 @@ def generate_index_s3(s3_client, bucket_name, prefix: str, upload=False):
         page_title = "ROCm SDK nightly tarballs"
     elif "prerelease" in bucket_lower:
         page_title = "ROCm SDK prerelease tarballs"
+    elif "release" in bucket_lower:
+        page_title = "ROCm SDK release tarballs"
```

Comment:
```suggestion
```
I'd say let's keep as is. `ROCm SDK tarballs` is fine for https://repo.amd.com/rocm/tarball/.